examples/joystick/Joystick.cpp

```diff
@@ -0,0 +1,176 @@
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics.hpp>
+#include <sstream>
+#include <iomanip>
+#include <string>
+#include <map>
+
+
+namespace
+{
+    typedef std::map<std::string, std::pair<sf::Text, sf::Text> > Texts;
+    Texts texts;
+    std::ostringstream sstr;
+
+    // Axes labels in as C strings
+    const char* axislabels[] = {"X", "Y", "Z", "R", "U", "V", "PovX", "PovY"};
```

Comment:
While it's unlikely the enum values for the axes might change, I'm not really a fan of just assuming a constant/fixed order here.

Reply:
In core SFML code I would agree, but this is just an example, so it's a good thing if we keep this kind of boilerplate code as short as possible.